apps/condo/domains/billing/schema/resolvers/propertyResolver.js

```diff
@@ -49,19 +50,34 @@ class PropertyResolver extends Resolver {
         if (loaded) {
             return loaded
         }
+
+        const knownMissingProperty = this.missingOrganizationPropertiesIndex[addressKey]
+        if (knownMissingProperty) {
+            return loaded
+        }
+
         const [property] = await find('Property', { addressKey, organization: { id: this.organizationId }, deletedAt: null })
         if (property) {
             this.organizationProperties.push(property)
+        } else {
+            this.missingOrganizationPropertiesIndex[addressKey] = true
         }
         return property
     }
 
     async init () {
+        // We need to load properties cache in case we do not use addressService for address normalize
+        // Let's select first 5000 properties - will take 15 mb approx to store them
+        // Exception is isCottageVillage - we have to load all properties for such organisations
+        const firstCondition = this.isCottageVillage ? {} : { first: 5000 }
+        this.organizationProperties = await itemsQuery('Property', {
```

Comment:
we do not need to load such cache. In case of small organizations - we will have 2-10 properties. In case of service providers - we will have 30K-100K properties

Reply:
99 percent of organizations has less than 5000 properties. So such cache going to be work for 99% percent cases

Reply:
I just run load test with pre created 1000 properties. Looks like this caching going to increate time to proceed a request. Will revert this change